crates/e2e/tests/e2e/limit_orders.rs

```diff
@@ -291,6 +297,147 @@ async fn two_limit_orders_test(web3: Web3) {
     .unwrap();
 }
 
+async fn two_limit_orders_multiple_winners_test(web3: Web3) {
+    let mut onchain = OnchainComponents::deploy(web3).await;
+
+    let [solver1, solver2] = onchain.make_solvers(to_wei(1)).await;
```

Comment:
nit to keep the naming consistent.
```suggestion
    let [solver_a, solver_b] = onchain.make_solvers(to_wei(1)).await;
```